src/features/world/scenes/BaseScene.ts

```diff
@@ -719,6 +721,9 @@ export abstract class BaseScene extends Phaser.Scene {
   }
 
   update(): void {
+    // mute audio if audio is muted
+    this.sound.mute = this.isAudioMuted;
```

Comment:
Is there a way we can get around calling this on everything update?

Maybe we could set it only if the value is different?